app/portainer/views/stacks/edit/stack.html

```diff
@@ -185,7 +185,7 @@
                     <button
                       type="button"
                       class="btn btn-sm btn-primary"
-                      ng-disabled="state.actionInProgress || !stackUpdateForm.$valid || !stackFileContent || orphaned"
+                      ng-disabled="state.actionInProgress || stack.Status === 2 || !stackFileContent || orphaned"
```

Comment:
1. why remove form validation?
2. stack.Status === 2 was removed to enable the update of stopped stacks. is there a reason to bring it back?